pkgs/by-name/ra/radarr/package.nix

```diff
@@ -0,0 +1,204 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  buildDotnetModule,
+  dotnetCorePackages,
+  sqlite,
+  withFFmpeg ? true, # replace bundled ffprobe binary with symlink to ffmpeg package.
+  servarr-ffmpeg,
+  fetchYarnDeps,
+  yarn,
+  fixup-yarn-lock,
+  nodejs,
+  nixosTests,
+  # update script
+  writers,
+  python3Packages,
+  nix,
+  prefetch-yarn-deps,
+  fetchpatch,
+  applyPatches,
+}:
+let
+  version = "5.25.0.10024";
+  # The dotnet8 compatibility patches also change `yarn.lock`, so we must pass
+  # the already patched lockfile to `fetchYarnDeps`.
+  src = applyPatches {
+    src = fetchFromGitHub {
+      owner = "Radarr";
+      repo = "Radarr";
+      tag = "v${version}";
+      hash = "sha256-Pzdkm9oxPJRdzdCjRC22XgLBS5ffBY5u9m2QwkoZzOM=";
+    };
+    patches =
+      [
+        ./nuget-config.patch
```

Comment:
I think doing the move as a postPatch script instead of applying it as a patch file is easier to understand, i.e.

```
postPatch = ''
  mv src/NuGet.config NuGet.Config
'';
```